tester_app/client/app/bundles/HelloWorld/store/helloWorldStore.jsx

```diff
@@ -0,0 +1,24 @@
+import { compose, createStore, applyMiddleware, combineReducers } from 'redux';
+import thunkMiddleware from 'redux-thunk';
+import loggerMiddleware from 'lib/middlewares/loggerMiddleware';
+import reducers from '../reducers';
+import { initalStates } from '../reducers';
+
+export default props => {
+  const initialName = props.initialName;
+  const { $$helloWorldState } = initalStates;
+  const initialState = {
```

Comment:
@justin808 Yeah I think that's necessary to say because intuitively I would have thought this needed to be an immutable. Added as this `// Redux expects to initialize the store using an Object, not an Immutable.Map`